integration/tool/README.md

```diff
@@ -0,0 +1,7 @@
+This part will contain many useful tools
```

Comment:
there is already an `integration/tools` directory so it is confusing to add an `integration/tool` directory. can you move these two files into `integration/tools/ratis-shell/`?

Reply:
done